cpp/src/arrow/compute/api_vector.h

```diff
@@ -174,6 +174,23 @@ class ARROW_EXPORT SelectKOptions : public FunctionOptions {
   std::vector<SortKey> sort_keys;
 };
 
+/// \brief Fetch options
+class ARROW_EXPORT FetchOptions : public FunctionOptions {
+ public:
+  explicit FetchOptions(int64_t offset = 0, int64_t count = 0,
+                        std::vector<SortKey> sort_keys = {}, bool sort_first = true);
+  static constexpr char const kTypeName[] = "FetchOptions";
+
+  /// The number of `offset` elements to skip.
+  int64_t offset;
+  /// The number of `count` elements to select.
+  int64_t count;
+  /// Column key(s) to order by and how to order by these sort keys.
```

Comment:
Do I have to specify sort keys?

Reply:
Yes.

Reply:
WDYT?

Reply:
Btw, the default is no sort_keys. You specify it only if you need sorting. By default it is not sorted if the keys are not provided.